xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1171 @@
+     images. The final path will be something like <filename>/tmp/skopeodata/registry.suse.com/</filename>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Populate the local registry with the downloaded data.
+    </para>
+    <para>
+     For <literal>--dest-creds</literal> you must use the credentials you
+     created during <xref linkend="sec.deploy.scenarios.airgap.container-registry.mirror" />.
+    </para>
+<screen>&prompt.user;<command>skopeo sync --dest-creds <replaceable>USERNAME:PASSWORD</replaceable> \
+dir:<replaceable>/tmp/skopeodata/registry.suse.com/</replaceable> <replaceable>docker://mymirror.local:5000</replaceable></command>
+     </screen>
+   </step>
+   <step>
+    <para>
+     After the synchronization is done, you can remove the <filename>skopeodata</filename>
+     directory.
+    </para>
+<screen>&prompt.user;<command>rm -rf /tmp/skopeodata</command></screen>
+   </step>
+  </procedure>
+
+   <procedure>
+    <title>Transfer Data To Secure Storage</title>
+    <step>
+     <para>
+      Connect the trusted storage to the external mirror.
+     </para>
+    </step>
+    <step>
+     <para>
+      Transfer the container image data to the trusted storage. This will remove
+      all files and directories that are no longer present on the external host
+      from the trusted storage.
+     </para>
+<screen>&prompt.user;<command>rsync -aP /tmp/skopeodata/registry.suse.com/ <replaceable>/mnt/storage/registry/</replaceable> --delete</command>
+     </screen>
+    </step>
+    <step>
+     <para>
+      Transfer the helm chart data to the trusted storage.
+     </para>
+<screen>&prompt.user;<command>rsync -aP <replaceable>/tmp/charts/</replaceable> <replaceable>/mnt/storage/charts</replaceable> --delete</command>
+     </screen>
+    </step>
+   </procedure>
+
+   <procedure>
+    <title>Update Internal Mirror</title>
+    <step>
+     <para>
+      Connect the trusted storage to the internal mirror.
+     </para>
+    </step>
+    <step>
+     <para>
+      Transfer the container image data to the internal mirror. This will remove
+      all files and directories that are no longer present on the trusted storage
+      from the internal mirror.
+     </para>
+     <para>
+      The target directory is <filename>/var/lib/registry</filename>.
+     </para>
+<screen>&prompt.user;<command>rsync -aP <replaceable>/mnt/storage/registry/</replaceable> <replaceable>/var/lib/registry/</replaceable> --delete</command>
+     </screen>
+    </step>
+    <step>
+     <para>
+      Transfer the helm chart data to the internal mirror. This will remove
+      all charts that do not exist on the trusted storage. If you have added
+      any charts to the location manually, please back up these first and
+      restore after the sync from the trusted storage is done.
+     </para>
+<screen>&prompt.user;<command>rsync -aP <replaceable>/mnt/storage/charts</replaceable> <replaceable>/srv/www/htdocs/charts/</replaceable> --delete</command>
```

Comment:
**DANGER:** this command will not work as expected. rsync is a PITA with trailing `/` inside of the source. The command above will lead to the creation of `/srv/www/htdocs/charts/charts/...`.

The right command is:

```
screen>&prompt.user;<command>rsync -aP <replaceable>/mnt/storage/charts/</replaceable> <replaceable>/srv/www/htdocs/charts/</replaceable> --delete</command>
```